purchases/src/main/kotlin/com/revenuecat/purchases/VirtualCurrencyInfo.kt

```diff
@@ -0,0 +1,34 @@
+package com.revenuecat.purchases
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * A class representing information about a virtual currency in the app.
+ *
+ * @property balance The current balance of the virtual currency.
+ */
+@Parcelize
+data class VirtualCurrencyInfo internal constructor(
+    val balance: Long,
```

Comment:
Just to check, we are not planning on supporting decimal numbers for the currencies correct?